### ISSUE TITLE
Can't save service config for custom script

### ISSUE THREAD

**Issue by DawoonC** [dreamfactorysoftware/dreamfactory#90]
```
Admin Application Version: 2.2.1
DreamFactory Version: 2.2.0
System Database: sqlite
```

I'm trying out newly released DF 2.2 on an ec2 instance and it seems there is a javascript error when I try to save script in service config page.
Below is the error message I got on my browser console (Chrome 51.0.2704.103, OS X)

```
vendor.fd6a5200.js:5 TypeError: Cannot set property 'content' of null
    at assign (vendor.fd6a5200.js:6)
    at Object.<anonymous> (vendor.fd6a5200.js:4)
    at k.$digest (vendor.fd6a5200.js:5)
    at k.$apply (vendor.fd6a5200.js:5)
    at Array.<anonymous> (app.a66ba06d.js:8)
    at b.r._emit.r._dispatchEvent (ace.js:1)
    at b.onBlur (ace.js:1)
    at HTMLTextAreaElement.<anonymous> (ace.js:1)(anonymous function) @ vendor.fd6a5200.js:5(anonymous function) @ vendor.fd6a5200.js:5(anonymous function) @ app.a66ba06d.js:11$digest @ vendor.fd6a5200.js:5$apply @ vendor.fd6a5200.js:5(anonymous function) @ app.a66ba06d.js:8r._emit.r._dispatchEvent @ ace.js:1onBlur @ ace.js:1(anonymous function) @ ace.js:1
vendor.fd6a5200.js:5 TypeError: Cannot read property 'config' of null
    at http://x.x.x.x/dreamfactory/dist/scripts/app.a66ba06d.js:5:4554
    at Array.forEach (native)
    at k (http://x.x.x.x/dreamfactory/dist/scripts/app.a66ba06d.js:5:4497)
    at k.a._updateService (http://x.x.x.x/dreamfactory/dist/scripts/app.a66ba06d.js:5:5140)
    at k.a.saveService (http://x.x.x.x/dreamfactory/dist/scripts/app.a66ba06d.js:5:3062)
    at http://x.x.x.x/dreamfactory/dist/scripts/vendor.fd6a5200.js:6:30852
    at http://x.x.x.x/dreamfactory/dist/scripts/vendor.fd6a5200.js:7:6018
    at k.$eval (http://x.x.x.x/dreamfactory/dist/scripts/vendor.fd6a5200.js:5:27109)
    at k.$apply (http://x.x.x.x/dreamfactory/dist/scripts/vendor.fd6a5200.js:5:27387)
    at HTMLFormElement.<anonymous> (http://x.x.x.x/dreamfactory/dist/scripts/vendor.fd6a5200.js:7:6000)(anonymous function) @ vendor.fd6a5200.js:5(anonymous function) @ vendor.fd6a5200.js:5(anonymous function) @ app.a66ba06d.js:11$apply @ vendor.fd6a5200.js:5(anonymous function) @ vendor.fd6a5200.js:7dispatch @ vendor.fd6a5200.js:2q.handle @ vendor.fd6a5200.js:2
vendor.fd6a5200.js:5 TypeError: Cannot set property 'content' of null
    at assign (vendor.fd6a5200.js:6)
    at Object.<anonymous> (vendor.fd6a5200.js:4)
    at k.$digest (vendor.fd6a5200.js:5)
    at k.$apply (vendor.fd6a5200.js:5)
    at HTMLFormElement.<anonymous> (vendor.fd6a5200.js:7)
    at HTMLFormElement.dispatch (vendor.fd6a5200.js:2)
    at HTMLFormElement.q.handle (vendor.fd6a5200.js:2)
```

**Comment by jimijon** [comment]
Yes, I had to first paste in the config before saving. Then I could save afterwards like normal.

**Comment by DawoonC** [comment]
@jimijon It seems the config can't be modified if it was created empty.
As you mentioned, it works after I created a new service with non-empty config.

I assume the save button does not bind correctly when config content is null.

**Comment by drewpearce** [comment]
known bug.
for internal documentation purposes, this is DF-778.
Currently being worked on. MIGHT be included in the upcoming release.

**Comment by vinodhum** [comment]
Do we have any work around for this issue  ?

**Comment by todda48** [comment]
The easiest thing to do is to delete the service and recreate it.  We'll try to get this fixed in Aug release. If not, in next one for sure.

**Comment by vinodhum** [comment]
ok found the solution for this.

while creating service make sure you don't save it before adding some config in config tab
just add some dummy code you can edit it later. if you create script service without adding any config you will get " Cannot set property 'content' of null "
hope this help.

**Comment by drewpearce** [comment]
This was resolved in the DF 2.3 release
